packages/cli/yaml/validator/src/rules/valid-oauth/valid-oauth.ts

```diff
@@ -11,6 +11,8 @@ import { Rule } from "../../Rule";
 import { CASINGS_GENERATOR } from "../../utils/casingsGenerator";
 import { validateClientCredentials } from "./validateClientCredentials";
 
+export const OAUTH_DOCS_LINK = "https://buildwithfern.com/learn/api-definition/fern/api-yml-reference#authentication";
```

Comment:
we've talked a lot about linking folks out to `docs` -- one thing that could make this more manageable to ensure we don't break is to create a top level `docsLinks.ts` 

```ts
export interface DocsLinks {
  oauth: string;
  ....
}

expornt const DocsLinks: DocsLinks {
   oauth: "https://", 
}
```

More parts of the code base can then import this utility and rely on it